tests/tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Streisand syntax check.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+export ANSIBLE_CONFIG=$DIR/ansible.cfg
+
+set -e
+
+command -v ansible > /dev/null 2>&1 || {
+  echo "Ansible is not installed."
+  exit 1
+}
+
+required_ansible_version="2.3.0.0"
+
+if [[ "$(ansible --version | grep -oe '2\(.[0-9]\)*')" < $required_ansible_version ]]
+then
+    echo "Ansible $required_ansible_version or higher is required."
+    exit 1
+fi
```

Comment:
Hmm ill have to think about the best way to do that.  I was considering doing something like

```- name: Validate ansible version
  assert:
    that:
      - ansible_version is defined
      - ansible_version.full|version_compare('2.3.0.0', '>=')
    msg: "You must have ansible >= 2.3.0.0 to run this specific playbook"
  run_once: yes```

and taking it out from both scripts.. but we would want the version check to happen before people run just about anything.